src/ContainerDetails.jsx

```diff
@@ -43,6 +44,17 @@ const ContainerDetails = ({ container }) => {
                         <DescriptionListDescription>{utils.quote_cmdline(container.Config.Cmd)}</DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    {Boolean(container.Config?.Labels?.PODMAN_SYSTEMD_UNIT) && (container.uid === 0 || container.uid === null) &&
```

Comment:
OOI, is that `Boolean()` wrapper for type checking? (we don't currently do that in podman)

Reply:
Ah more a copy-pasta from `const isSystemdService = Boolean(container.Config?.Labels?.PODMAN_SYSTEMD_UNIT);` not type related.

Reply:
It's fine, I was just curious.